modules/config-fields-misc.adoc

```diff
@@ -94,4 +94,8 @@
   + 
 **Default:** False
 
+|*DISABLE_PUSHES* |Boolean | Disables pushes of new content to the registry while retaining all other functionality. Differs from `read-only` mode because database is not set as `read-only`. When `DISABLE_PUSHES` is set to `true`, the {productname} garbage collector is disabled. As a result, when `PERMANENTLY_DELETE_TAGS` is enabled, using the {productname} UI to permanently delete a tag does not result in the immediate deletion of a tag. Instead, the tag stays in the repository until `DISABLE_PUSHES` is set to `false`, which re-enables the garbage collector. {productname} administrators should be aware of this caveat when using `DISABLE_PUSHES` and `PERMANENTLY_DELETE_TAGS` together. +
```

Comment:
"the tag stays in the repository until" better to -> "the image stays in the `storage` or (`backend storage`) until".
it does delete the tag on UI, user can't see it in the repository, it should be removed from repository but not clean from backend storage

Reply:
Fixed.